pkgs/tools/text/dfmt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, python3
+}:
+
+let
+  inherit (python3.pkgs)
+    buildPythonApplication
+    fetchPypi
+    pythonOlder;
+in
```

Comment:
```suggestion
with python3.pkgs;
```

Reply:
I'm not a fan of global-scope `with` statements. I'd rather be explicit.